vmm/src/default_syscalls/x86_64.rs

```diff
@@ -163,6 +165,19 @@ pub fn default_context() -> Result<SeccompFilterContext, Error> {
                 libc::SYS_exit_group,
                 (0, vec![SeccompRule::new(vec![], SeccompAction::Allow)]),
             ),
+            (
+                libc::SYS_fcntl,
```

Comment:
Add `SYS_fcntl` to the `ALLOWED_SYSCALLS` array too in case anyone wants to use seccomp with level=1.